src/app/reporting/SynchronizedReportSchedulerImpl.cpp

```diff
@@ -188,13 +182,14 @@ void SynchronizedReportSchedulerImpl::TimerFired()
     mNodesPool.ForEachActiveObject([now, &firedEarly](ReadHandlerNode * node) {
         if (node->GetMinTimestamp() <= now)
         {
+            // Mark the handler as CanBeSynced if the min interval has elapsed so it will emit a report on the next engine run
```

Comment:
This might be clearer as:

> Since this handler can now report whenever it wants to, mark it as allowed to report if any other handler is reporting.

or so.  That's the real intent of what's going on here, right?

Reply:
Yes it is.